operator/pkg/apis/controlplane/v1alpha1/controlplane.go

```diff
@@ -46,9 +46,10 @@ type ControlPlaneList struct {
 // master and etcd are configured to run. By default, KIT uses all the default
 // values and ControlPlaneSpec can be empty.
 type ControlPlaneSpec struct {
-	KubernetesVersion string     `json:"kubernetesVersion,omitempty"`
-	Master            MasterSpec `json:"master,omitempty"`
-	Etcd              Etcd       `json:"etcd,omitempty"`
+	KubernetesVersion         string     `json:"kubernetesVersion,omitempty"`
+	ColocateAPIServerWithEtcd bool       `json:"colocateAPIServerWithEtcd,omitempty"`
```

Comment:
what's the use-case for this ?

Reply:
There are scenarios where we just want to test the functionality of a component and don't need a 4-5 node cluster.